proto/text.go

```diff
@@ -551,9 +558,22 @@ func (tm *TextMarshaler) writeAny(w *textWriter, v reflect.Value, props *Propert
 		// Other values are handled below.
 	}
 
+	// Handle Uints
```

Comment:
What does this mean actually? Was it that before this PR, printing an Any with an uint inside would panic?